libraries/src/Console/AddUserCommand.php

```diff
@@ -238,48 +238,95 @@ protected function getUserGroups(): array
 
         $groupList = [];
 
-        // Group names have been supplied as input arguments
+        // Group values have been supplied
         if (!\is_null($groups) && $groups[0]) {
             $groups = explode(',', $groups);
 
-            foreach ($groups as $group) {
-                $groupId = $this->getGroupId($group);
+            foreach ($groups as $groupInput) {
+                $groupInput = trim($groupInput);
+
+                // Try ID
+                if (ctype_digit($groupInput)) {
+                    $groupId = $this->getGroupIdById((int) $groupInput);
+                } else {
+                    $groupId = $this->getGroupIdByName($groupInput);
+                }
 
                 if (empty($groupId)) {
-                    $this->ioStyle->error("Invalid group name '" . $group . "'");
-                    throw new InvalidOptionException("Invalid group name " . $group);
+                    $this->ioStyle->error("Invalid group: '" . $groupInput . "'");
+                    throw new InvalidOptionException("Invalid group: " . $groupInput);
                 }
 
-                $groupList[] = $this->getGroupId($group);
+                $groupList[] = $groupId;
             }
 
             return $groupList;
         }
 
-        // Generate select list for user
+        // No group provided → prompt interactively
         $query = $db->getQuery(true)
             ->select($db->quoteName('title'))
             ->from($db->quoteName('#__usergroups'))
             ->order($db->quoteName('id') . 'ASC');
         $db->setQuery($query);
-
         $list = $db->loadColumn();
 
         $choice = new ChoiceQuestion(
-            'Please select a usergroup (separate multiple groups with a comma)',
+            'Please select a usergroup (separate multiple groups with comma)',
             $list
         );
         $choice->setMultiselect(true);
 
         $answer = (array) $this->ioStyle->askQuestion($choice);
 
         foreach ($answer as $group) {
-            $groupList[] = $this->getGroupId($group);
+            $groupList[] = $this->getGroupIdByName($group);
         }
 
         return $groupList;
     }
 
+    /**
+     * @param   string  $groupName
+     *
+     * @return  int|null
+     * 
```

Comment:
```suggestion
     *
```
Fix PHPCS failing in Drone.